routes/session.js

```diff
@@ -118,7 +118,7 @@ function sessionRoutes (server, options, next) {
           })
         }
 
-        throw error
+        // throw error
```

Comment:
Coudl remove the change and `git push --force` with the cleaned up commits? If not I can do that for you